2013-09-22 18:48:45,891 [   INFO] Forge tools running at version 3.3.64
2013-09-22 18:48:45,891 [  DEBUG] Checking for update zip...
2013-09-22 18:48:45,896 [  ERROR] No folder called 'development' found. You're trying to run your app but you haven't built it yet!
Try forge build first.
2013-09-22 18:48:45,896 [  DEBUG] Traceback (most recent call last):
  File "/Users/gilesbutler/Library/Trigger Toolkit/build-tools/forge/async.py", line 106, in run
    result = self._target(*self._args, **self._kwargs)
  File "/Users/gilesbutler/Library/Trigger Toolkit/build-tools/forge/main.py", line 422, in run
    _assert_have_development_folder()
  File "/Users/gilesbutler/Library/Trigger Toolkit/build-tools/forge/main.py", line 63, in _assert_have_development_folder
    raise ForgeError(message)
ForgeError: No folder called 'development' found. You're trying to run your app but you haven't built it yet!
Try forge build first.

2013-09-23 14:49:38,165 [   INFO] Forge tools running at version 3.3.64
2013-09-23 14:49:38,165 [  DEBUG] Checking for update zip...
2013-09-23 14:49:38,170 [  ERROR] Source folder "src" does not exist - have you run forge create yet?
2013-09-23 14:49:38,170 [  DEBUG] Traceback (most recent call last):
  File "/Applications/TriggerToolkit.app/Contents/MacOS/build-tools/forge/async.py", line 106, in run
    result = self._target(*self._args, **self._kwargs)
  File "/Applications/TriggerToolkit.app/Contents/MacOS/build-tools/forge/main.py", line 308, in development_build
    prog=ENTRY_POINT_NAME,
ForgeError: Source folder "src" does not exist - have you run forge create yet?

